app/assets/javascripts/components/instructor_panel.js.jsx

```diff
@@ -44,10 +58,12 @@ var InstructorPanel = React.createClass({
       <div className={this.props.segmentClass}>
         <h4 className="ui header">Instructor Queue Management</h4>
 
+        <Action data={this.getQueuePopandPinButtonData()} />
```

Comment:
Thoughts on putting this below the "queue pop" button so instructors who'd prefer to keep their existing workflow don't have to re-train their muscle memory?

Reply:
That seems like a good idea!